gulp-core-build-karma/karma.config.js

```diff
@@ -4,11 +4,14 @@
 // Generated on Thu Oct 08 2015 18:13:05 GMT-0700 (PDT)
```

Comment:
This comment isn't really relevant anymore.

And this file is pretty complicated. Can we open a work item to convert it to TS?